serve/mlc_serve/model/paged_cache_model.py

```diff
@@ -505,6 +517,9 @@ def generate(
                 )
 
                 if maybe_new_token is not None:
+                    if not maybe_new_token in requests[i].sampling_params.appeared_tokens_freq:
+                        requests[i].sampling_params.appeared_tokens_freq[maybe_new_token] = 0
+                    requests[i].sampling_params.appeared_tokens_freq[maybe_new_token] += 1
```

Comment:
I think `maybe_new_token` is not an int, `maybe_new_token[0]` is. `maybe_new_token` is an array with one element.

Reply:
I see, just fixed.